Eos/Soave-Redlich-Kwong/EOS.H

```diff
@@ -496,37 +479,109 @@ RYET2P(
   amrex::Real& T,
   amrex::Real& P)
 {
-  int lierr = 0;
-  GET_T_GIVEN_EY(&E, Y, &T, &lierr);
-  CKPY(&R, &T, Y, &P);
+  // Note: this function was written to have the same behavior as the Fuego
+  // version (T gets overwritten, so the function is actually RYE2TP)
+  // I'm not sure if this behavior makes sense. The function is only called
+  // once in PeleC, in pc_derpmmserror. One MixingRuleAm could be eliminated
+  // by expanding these function explicitly, but that's small relative to
+  // the overall cost. By using T as in input (unlike Fuego) iteration would
+  // be eliminated and the overall cost would be greatly reduced. - BAP
+  EOS::REY2T(R,E,Y,T);
+  EOS::RTY2P(R,T,Y,P);
```

Comment:
On one hand, the usual comment applies: only waste time on it if it shows up in a profile as significant.  That said, I don't think there's a ton of value in fighting for maximum consistency between the EOS calls.  We can also ifdef code selection based on EOSand make it a compile time decision.

Reply:
Yeah how this is handled may change eventually. The best strategy may be to just rename the function to RYE2TP so the name better reflects the behavior for each EoS. But right now it seems like a non-issue, so it seems like leaving it in the present form is reasonable.